lib/gettext-setup/pot.rb

```diff
@@ -71,12 +71,11 @@ def self.generate_new_pot(locales_path = GettextSetup.locales_path, path = nil)
 
     def self.generate_new_po(language, locales_path = GettextSetup.locales_path,
                              pot_file = nil, po_file = nil)
+      language ||= ENV['LANGUAGE']
```

Comment:
@Magisus Actually, I don't see an obvious way of doing this without making changes that might have side-effects in bits of the code I haven't had a chance to look at